cmd/podman/README.md

```diff
@@ -1,15 +1,113 @@
-It is intended as a counterpart to CRI-O, to provide low-level debugging not available through the CRI interface used by Kubernetes.
-It can also act as a container runtime independent of CRI-O, creating and managing its own set of containers.
-
-## Use cases
-1. Create containers
-2. Start, stop, signal, attach to, and inspect existing containers
-3. Run new commands in existing containers
-4. Push and pull images
-5. List and inspect existing images
-6. Create new images by committing changes within a container
-7. Create pods
-8. Start, stop, signal, and inspect existing pods
-9. Populate pods with containers
+# Adding a podman V2 commands
+
+## Build podman V2
+
+```shell script
+$ cd $GOPATH/src/github.com/containers/libpod/cmd/podmanV2
+```
+If you wish to include the libpod library in your program,
+```shell script
+$ go build -tags 'ABISupport' .
+```
+The `--remote` flag may be used to connect to the Podman service using the API.
+Otherwise, direct calls will be made to the Libpod library.
+```shell script
+$ go build -tags '!ABISupport' .
+```
+The Libpod library is not linked into the executable.
+All calls are made via the API and `--remote=False` is an error condition.
+
+## Adding a new command `podman manifests`
+```shell script
+$ mkdir -p $GOPATH/src/github.com/containers/libpod/cmd/podmanV2/manifests
+```
+Create the file ```$GOPATH/src/github.com/containers/libpod/cmd/podmanV2/manifests/manifest.go```
+```go
+package manifests
+
+import (
+    "github.com/containers/libpod/cmd/podman/registry"
+    "github.com/containers/libpod/pkg/domain/entities"
+    "github.com/spf13/cobra"
+)
+
+var (
+    // podman _manifests_
+    manifestCmd = &cobra.Command{
+        Use:               "manifest",
+        Short:             "Manage manifests",
+        Long:              "Manage manifests",
+        Example:           "podman manifests IMAGE",
+        TraverseChildren:  true,
+        PersistentPreRunE: preRunE,
+        RunE:              registry.SubCommandExists, // Report error if there is no sub command given
+    }
+)
+func init() {
+    // Subscribe command to podman
+    registry.Commands = append(registry.Commands, registry.CliCommand{
+        // _podman manifest_ will support both ABIMode and TunnelMode
+        Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+        // The definition for this command
+        Command: manifestCmd,
+    })
+    // Setup cobra templates, sub commands will inherit
+    manifestCmd.SetHelpTemplate(registry.HelpTemplate())
+    manifestCmd.SetUsageTemplate(registry.UsageTemplate())
+}
+
+// preRunE populates the image engine for sub commands
+func preRunE(cmd *cobra.Command, args []string) error {
+    _, err := registry.NewImageEngine(cmd, args)
+    return err
+}
+```
+To "wire" in the `manifest` command, edit the file ```$GOPATH/src/github.com/containers/libpod/cmd/podmanV2/main.go``` to add:
```

Comment:
nit, wouldn't this be 
`Create the file ```$GOPATH/src/github.com/containers/libpod/cmd/podman/manifests/inspect.go`
now?

Reply:
yeah

Reply:
we take prs

Reply:
but you're so good at them :smiley: